src/KeyworkerProfile/container.js

```diff
@@ -13,10 +14,15 @@ class KeyworkerProfileContainer extends Component {
     this.displayError = this.displayError.bind(this);
     this.getKeyworkerList = this.getKeyworkerList.bind(this);
     this.gotoKeyworkerResults = this.gotoKeyworkerResults.bind(this);
+    this.gotoKeyworkerProfile = this.gotoKeyworkerProfile.bind(this);
   }
 
   componentWillMount () {
-    this.props.setCurrentPageDispatch(1);
+    if (this.props.paramStaffId) {
+      this.gotoKeyworkerProfile(this.props.paramStaffId);
```

Comment:
an approach for deep linking and linking from a list of results - can discuss

Reply:
Looks good